backend/data.go

```diff
@@ -109,7 +109,7 @@ type DataResponse struct {
 	// The data returned from the Query. Each Frame repeats the RefID.
 	Frames data.Frames
 
-	// Error is a property to be set if the the corresponding DataQuery has an error.
+	// Error is a property to be set if the corresponding DataQuery has an error.
 	Error error
```

Comment:
So does this property become deprecated? We want to avoid confusion about which of these should be used.

Reply:
Nope so this stays the same. The only different is what you set this property as. You can use `errors.New` or `fmt.Errorf` to your hearts content still. But if you use `backend.NewError` - this is how we can provide 1st class support in communicating back to a user. The rest, will rely a bit on guesswork (IE how it currently works)

Reply:
Nevermind, I thought this was a Frame property for a little bit.. to early.